docs/workflows/batch_processing/batch-processing.md

```diff
@@ -0,0 +1,56 @@
+# **Batch processing**
+
+Besides the interactive [GUI](../../main-window-overview.md), MZmine allows the user to run processing workflows in an automated manner using the "batch mode". Entire processing pipelines (including data import/export) can be run with few clicks, or even through the command-line application. This makes MZmine suitable to be integrated into automated data analysis pipelines (e.g. QC systems).
+
+Batch files (XML format) are essentially lists of tasks run by MZmine one after another. Any of the methods available in MZmine 3 can be included in the batch file.
+
+---
+## **How to run batch processing**
+
+**Project :material-arrow-right-thin: Batch mode**
+![batch-mode](batch_mode.png)
+
+When a new step is added to the queue its parameter setup dialog is shown. The "Set paramaters" button allows the user to modify a step's parameter settings. The "Clear" button removes all steps. The "Load" and "Save" buttons make it possible to read and write batch steps to XML files.
+
+The first step of a batch queue is performed on those raw data files and/or peak lists selected by the user. The remaining steps are performed on the results produced by each preceding step. For example, if the first step of the batch queue is Chromatogram builder, it will produce peak lists as a result. If the following step were Peak list deconvolution then it will be performed on the peak lists produced by the preceding Chromatogram builder step.
```

Comment:
The remaining steps are performed on the results produced by each preceding step **(File/Feature list selection must be set to _Those created by previous batch step_)** .